### ISSUE TITLE
Can't Create Note - 22.05 - actually it's a flatpak permission issue

### ISSUE THREAD

**Issue by kuiperzone** [toolstack/Folio#31]
I updated to the lastest flatpak using Gnome explorer. To do this, I first removed the previous version, which I had installed as a flatpak file download via the command line.

Now, can't now create new notes. Screenshot attached.

As I write this, I discovered the problem -- Folio needs the Flatpak permission to access the note storage location.

You see, I have multiple drives mounted under `/mnt`, and I can't be the only one with multiple drives where much of my stuff is not stored under home. For me, most flatpaks apps are unusable without Flatseal. :(

Anyway, see screenies. And thanks for the updated version. It's working now I have changed the permission in Flatseal. But something you want to consider in next version?

![image](https://github.com/toolstack/Folio/assets/17156238/7926df9b-2427-4fc5-8702-68d213ec244b)

![image](https://github.com/toolstack/Folio/assets/17156238/06347ca5-08bd-4587-9d39-1f601050ada9)

![image](https://github.com/toolstack/Folio/assets/17156238/ad228376-6710-4cb4-9a87-3e0f2243efc7)

**Comment by toolstack** [comment]
Yes, when I submitted to flathub they flagged the fact that the file permissions were for the entire host, which I agree is excessive.  I updated them to the home directory as anything less is probably too restrictive.

I suspect you're probably in the very small minority of users that have this kind of situation where you mount your data partitions outside of home and would run into this (happy to be proven wrong though).  Likewise, those that do this, are probably more tech savvy than most and just use flatseal to fix the issue 🤷

Unless I get a torrent of complaints about the permission I'll leave it as it is.
